cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -35,7 +35,7 @@
     <div class="o-form_group">
         <fieldset class="o-form_fieldset">
             <legend class="a-legend">
-                {{ value.was_it_helpful_text }}
+                {{ _(value.was_it_helpful_text) }}
```

Comment:
What will happen if someone provides alternate "was it helpful" text? Does the i18n default to a no-op, e.g. that text would be displayed as-is in English even on Spanish pages?

Reply:
I think this is intended to translate the `was it helpful` text used on the answer pages (["Was this answer helpful to you?"](https://github.com/cfpb/cfgov-refresh/blob/6307078b981b32e372c5caac0c3213ac3f2c088e/cfgov/locale/es/LC_MESSAGES/django.po#L96-L97)) for the Spanish pages & any other text should be output as entered.

Reply:
@chosak @virginiacc has it right: If any "was it helpful" text on a Spanish page does not exactly match "Was this answer helpful to you?" then no translation will occur. On English pages, translation will not occur regardless of what the 'helpful text is.